workflows/templates/ncn-reboot-common.yaml

```
@@ -0,0 +1,305 @@
+#
+# MIT License
+#
+# (C) Copyright 2025 Hewlett Packard Enterprise Development LP
+#
+# Permission is hereby granted, free of charge, to any person obtaining a
+# copy of this software and associated documentation files (the "Software"),
+# to deal in the Software without restriction, including without limitation
+# the rights to use, copy, modify, merge, publish, distribute, sublicense,
+# and/or sell copies of the Software, and to permit persons to whom the
+# Software is furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included
+# in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+# THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+# OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+# ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+# OTHER DEALINGS IN THE SOFTWARE.
+#
+apiVersion: argoproj.io/v1alpha1
+kind: WorkflowTemplate
+metadata:
+  name: ncn-reboot-common
+  namespace: argo
+spec:
+  templates:
+    - name: verify-pre-reboot-checks
+      inputs:
+        parameters:
+          - name: dryRun
+          - name: media_host
+            value: 'ncn-m001'
+      dag:
+        tasks:
+          - name: verify-pre-reboot-checks
+            templateRef:
+              name: ssh-template
+              template: shell-script
+            arguments:
+              parameters:
+                - name: dryRun
+                  value: "{{inputs.parameters.dryRun}}"
+                - name: mediaHost
+                  value: "{{inputs.parameters.media_host}}"                         
+                - name: scriptContent
+                  value: |
+                    echo "INFO NCN pre-reboot checks"
+
+                    echo "INFO Starting platform health check"
+                    result=$(/opt/cray/platform-utils/ncnHealthChecks.sh 2>&1)
+                    if [[ "$result" == *"FAILED"* ]]; then
+                      echo "ERROR platform health check has failed"
+                      exit 1
+                    else
+                      echo "INFO Successfully completed platform health check"
+                    fi
+                    
+                    echo "INFO Checking the status of the slurmctld pods"
```

There's only one pod
```suggestion
                    echo "INFO Checking the status of the slurmctld pod"
```